### DISCUSSION TITLE
Vercel servers not honoring "+" in URL

### DISCUSSION THREAD

**Discussion by Earl-Brown** [vercel/community#4565] [Bug Report]
### Page to Investigate

https://leadsheets.thegleep.com/+/10000%20reasons.crd

### Steps to Reproduce
1 - navigate to https://leadsheets.thegleep.com/+/10000%20reasons.crd - observe the "404" error
2 - navigate to https://leadsheets.thegleep.com/%2B/10000%20reasons.crd - observe "10000 reasons.crd" was downloaded

On the Vercel servers, this page yields a 404 error.  However on my local machine it works fine.

If I "encode" the "+" to "%2B", the page is found (and the file is downloaded)

This should be a static site, so no server-side processing involved.

My suspicion is some kind of routing rule that is blocking a non-encoded "+" (which is an acceptable character in a url).

I don't know how long this has been happening.

**Comment by amyegan** [comment]
I wasn't able to replicate this behavior in my test app. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use for debugging?

**Reply by Earl-Brown** [reply]
Let me see what I can come up with.

The repository for this specific situation is public:

https://github.com/Earl-Brown/leadsheets

It is a simple static site with a single build step that creates a single json representation of all the files in the project.

**Reply by Earl-Brown** [reply]
> I wasn't able to replicate this behavior in my test app. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use for debugging?

Have you made any progress on this issue?
